app.py

```diff
@@ -1,20 +1,135 @@
-    return render_template('index.html', items=items)
-    #return 'Hello World!'
+    url = "https://api.trello.com/1/boards/{id}/cards"
+    query = cf.get_trello_query()
+    
+    response = requests.request(
+        "GET",
+        url,
+        params=query
+        )
+    #print(response.text)
+    #items = session.get_items()
+    #return render_template('index.html', items=items)
+    cards = json.loads(response.text)
+    items_list = list()
+    for card in cards:
+        item = Item(card.id, card.status, card.title)
+        items_list.append(item)
 
-@app.route('/add', methods=['POST'])
+    return render_template('index.html', items=items_list)
+
+# To Be Removed
+# @app.route('/add', methods=['POST'])
 def addToDo():
     title = request.form['title']
     session.add_item(title)
     return redirect(url_for('index'))
 
 if __name__ == '__main__':
     app.run()
+
+
+"""
+Add a complete_item route that accepts the ID of an item as a
+parameter and then calls a method to change its status from 'To
+Do' to 'Done'
+curl -i "localhost:5000/api/foo?a=hello&b=world" 
+"""
+@app.route('/complete_item/<idCard>', methods=['PUT'])
+def update_card(idCard):
+
+    add_card_to_done()
+
+    url = "https://api.trello.com/1/cards/{idCard}"
+    headers = {"Accept": "application/json"}
+    query = cf.get_trello_query()
+    #No obvious way of retrieving the done List Id, 
+    #therefore reusing/mocking the example one instead 
+    query['idList'] = '5abbe4b7ddc1b351ef961415'
+    response = requests.request( "PUT", url, headers=headers, params=query )
+    print(json.dumps(json.loads(response.text), sort_keys=True, indent=4, separators=(",", ": ")))
+
+
+#create card/item
+@app.route('/add', methods=['POST'])
+def add_card(list_id):
+    url = "https://api.trello.com/1/cards"
+    
+    query = cf.get_trello_query()
+    #No obvious way of retrieving the created List Id, 
+    #therefore reusing/mocking the example one instead 
+    query['idList'] = list_id
+    response = requests.request("POST", url, params=query )
+    #print(response.text)
+    return redirect(url_for('index'))
+
+
+def add_card_to_todo():
+    add_card('todo_list_id')
+
+
+def add_card_to_done():
+    add_card('done_list_id')
+
+
+#delete card/item
+def remove_card(id):
+    url = f"https://api.trello.com/1/cards/{id}"
+    response = requests.request("DELETE", url )
+    print(response.text)
+
+
+#Fetch Cards on a list
+def get_cards(id):
+    url = f"https://api.trello.com/1/lists/{id}/cards"
+    
+    query = cf.get_trello_query()    
+    response = requests.request("GET", url, params=query)
+    print(response.text)
+
+
+# create lists
+def add_list_to_board(name):
+    url = "https://api.trello.com/1/lists"
+    query = cf.get_trello_query()
+    
+    #query['name'] = 'MyCorndelDevOpsToDoBoard'
+    query['name'] = name
+    #No obvious way of retrieving the created Board Id, 
+    #therefore reusing/mocking the example one instead 
```

Comment:
If you keep using a hardcoded id, that's fine but at least mention it in the README.

Alternatively, use an environment variable, or if you find the id by searching for a particular name, then that also needs to be in the README.